hacep/src/main/java/org/kie/hacep/Config.java

```diff
@@ -20,17 +20,13 @@
 import java.util.Map;
 import java.util.Properties;
 
-import org.kie.remote.CommonConfig;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class Config {
 
+    private static final Logger logger = LoggerFactory.getLogger(Config.class);
     public static final String BOOTSTRAP_SERVERS_KEY = "bootstrap.servers";
-    public static final String BATCH_SIZE_KEY = "batch.size";
```

Comment:
Seems as not related to documentation refactoring. Please create a separate PR with description of purpose for this change.

Reply:
Is a clean of unused configuration variables, I'll open a separate PR